src/Components/test/testassets/BasicTestApp/EventBubblingComponent.razor

```diff
@@ -1,10 +1,9 @@
 ﻿<h3 id="event-bubbling">Bubbling standard event</h3>
 
-@* Temporarily hard-coding the internal names - this will be replaced once the Razor compiler supports @onevent:stopPropagation and @onevent:preventDefault *@
 <div @onclick="@(() => LogEvent("parent onclick"))">
     @* This element shows you can stop propagation even without necessarily also handling the event *@
-    <div __internal_stopPropagation_onclick="@intermediateStopPropagation">
-        <button id="button-with-onclick" @onclick="@(() => LogEvent("target onclick"))" __internal_stopPropagation_onclick="@targetStopPropagation">
+    <div @onclick:stopPropagation="@intermediateStopPropagation">
+        <button id="button-with-onclick" @onclick="@(() => LogEvent("target onclick"))" @onclick:stopPropagation="@targetStopPropagation">
```

Comment:
The changes in this file are not strictly required for the PR, but I'm tidying up as I go